MM-30762. Updated Using Micetro documentation.

rachelbritton opened the pull request:
Updated the Using Micetro section of the user manual. This included renaming the sections for clarity/standardization, updating old images, and improving language and instructions.

rachelbritton committed:
MM-30762. Updated Using Micetro documentation.

Updated the Using Micetro section of the user manual. This included renaming the sections for clarity/standardization, updating old images, improving language and instructions.

valgeir commented:
Hey @rachelbritton 

For such large diffs, can we get a screenshot of the built documentation. Easier to read the end result than the diff between the two when it essentailly was a rewrite.

rachelbritton commented:
@valgeir, images of the built documentation in order here: 
![Using Micetro 1](https://github.com/user-attachments/assets/7a8e30ef-6993-40d1-89c0-651f077241d3)
![Using Micetro 2](https://github.com/user-attachments/assets/561057b3-d43c-4707-b335-4e585a2801e7)
![Using Micetro 3](https://github.com/user-attachments/assets/edd7f484-7112-4f2c-8c85-0ec323422bb1)
![Using Micetro 4](https://github.com/user-attachments/assets/d3679a38-475a-4962-9952-3c5e3c21fee1)
![Using Micetro 5](https://github.com/user-attachments/assets/6d8228f0-2e1f-46ab-8107-dd7c8490b768)
![Using Micetro 6](https://github.com/user-attachments/assets/63c33754-1ecc-430a-a46c-a3e5a8304c51)

valgeir commented on guides/user-manual/webapp_user_guide.rst:
```diff
@@ -7,99 +7,142 @@
 
 .. note::
-   The Internet Explorer browser is no longer supported as of version 9.3.0.
+   As of version 9.3, Micetro does not support the Internet Explorer browser.
```
I think we can just remove this. We are quite a bit away now from 9.3

valgeir commented on guides/user-manual/webapp_user_guide.rst:
```diff
@@ -7,99 +7,142 @@
-* **Admin**. The Admin page is the workspace for system administrators where they perform the various tasks to manage Micetro, such as managing user access and services.
+* **DNS**. On the :ref:`dns` page, you can create, delete, and edit DNS zones. You also manage DNS records within the DNS zones.
+* **IPAM**. On the :ref:`ipam` page, you have the ability to perform various actions related to network and DHCP management, such as creating a new IP address range or DHCP scope, managing IP addresses within a network or DHCP scope, and creating DHCP reserverations. You can also view related DNS data for IP addresses within a network and add DNS hosts for IP addresses.
+* **Reports**. The :ref:`webapp-reporting` page is activated with a separate license key. On this page, you can create and save new reporting definitions, as well as schedule reports to run at specific times.
```
Reporting is actually always active, but for creating your own report definitions to be re-used to scheduling reports you need the **Advanced Reporting Module**

valgeir commented on guides/user-manual/webapp_user_guide.rst:
```diff
@@ -7,99 +7,142 @@
-Micetro is centered around the grid listing objects in the system, such as DNS zones, DNS records, IP address ranges, DHCP scopes, and IP addresses. 
+Working in the grid
+-------------------
+Micetro is centered around a grid, which lists objects in the system, such as DNS zones and records, IP address ranges, DHCP scopes, and IP addresses. From the grid, users can manage DNS zones and records, networks and DHCP scopes, and more, using the tools that comprise the Micetro UI.
```
What do you think about using the term **Data grid** for the main content grids? There are other grids/tables in nested tasks but they behave differently from the main data grids such as in the DNS/IPAM/Service Management pages.

valgeir replied:
Potentially drop the UI part at the end of the sentence?

rachelbritton replied:
Yes to both. I'm going to add **Data grid** to the glossary, as well.

valgeir commented on guides/user-manual/webapp_user_guide.rst:
```diff
@@ -7,99 +7,142 @@
-The left-hand filtering sidebar acts as a selector in each context, and helps you to quickly find the zones or networks. You can select pre-defined “filters” or selectors.
+Filtering with the sidebar
+--------------------------
+The leftmost sidebar acts as a selector on each page, which helps you to quickly find the zones or networks. You can select predefined filters or selectors.
```
Do we think that it truly acts as a selector? I think it more is centered around these predefined filters for the data grids.

valgeir commented on images/web-UI-annotated.jpg:
For the toggles, they are called **Flat** and **Tree** views.

rachelbritton committed:
MM-30762. Revised text and changed image.

Revised text based on feedback and updated the annotated UI image with accessible colors and correct terminology.
